modules/visual_script/doc_classes/VisualScript.xml

```diff
@@ -26,23 +26,23 @@
 			</return>
 			<argument index="0" name="name" type="StringName">
 			</argument>
+			<argument index="1" name="arg1" type="int">
```

Comment:
arg1 is invalid

Reply:
Yes, that means that you didn't give a name to the argument in the `bind_method` call.